src/utils/processingQueue.ts

```diff
@@ -55,8 +55,7 @@ export class ProcessingQueue<T> {
           .catch((e) => {
             this.#logger?.(LogType.system, 'Event was not processed:', e);
             this.#catch(e);
-          })
-          .finally();
```

Comment:
If provided `_catch` throws exception, the message processing will stop, but it should not.

Reply:
I asked in an internal TypeScript chat (before removing it) and people said this is a no-op. Let me double check.

Reply:
Since this is the only blocker, for now I re-added the `.finally()` just so that we can submit this PR before the weekend.

Reply:
Oh, you are right! `finally` does not change the state of promise, so another catch is needed there. Out of scope of this PR of cause.